src/components/Settings/Settings.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import style from './Settings.module.css'
```

Comment:
You forgot a new line above style import.

Reply:
/fxd